### DISCUSSION TITLE
Related to contribute in your project?

### DISCUSSION THREAD

**Discussion by priyanshu-kun** [preactjs/preact#3090] [General]
Hey! I want to contribute in your project but your code base is quite large and I don't able to understand where should I start looking. But I really want to become a part of your community can anybody guide me if he/she doesn't mind or have time to guide me

I hope you will accept a newbie in your project like me.
Thank you very much in advance...

**Comment by nothinginterested** [comment]
just fork it and start to debug

**Comment by priyanshu-kun** [comment]
where should I start to debug this project?

**Comment by marvinhagemeister** [comment]
We have a file with quick introduction for contributors in [our repository](https://github.com/preactjs/preact/blob/master/CONTRIBUTING.md). After reading that a good step is to go through the issues list, pick one which interests you or where you feel like you can solve that and file a PR 👍
